README.md

```diff
@@ -17,6 +17,10 @@ More design details can be explored in the [WMCO enhancement](https://github.com
 The operator can be installed from the *community-operators* catalog on OperatorHub.
 It can also be build and installed from source manually, see the [development instructions](docs/HACKING.md).
 
+The operator must be deployed in the `openshift-windows-machine-config-operator` namespace for it to function properly.
```

Comment:
@mburke5678 in the OpenShift docs in addition to above, I suggest adding a known upgrade issue. Something along the lines of:
```
If the operator has been installed in a namespace other than `openshift-windows-machine-config-operator`, upgrading to 7.0.0 will result in failure when adding a node or bringing up the operator (TBD). Workaround is to reinstall the operator. Workloads will not be affected.
```

We should then backport the installation update to other releases.